### ISSUE TITLE
A11y_Visual Studio Code Client_Sep2021_Debug Adapter Protocol_Home Page_Screenreader: Main landmark is missing.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/debug-adapter-protocol#212]
**Environment Details:**

Application Name: Visual Studio Code Client
#URL: https://microsoft.github.io/debug-adapter-protocol
Browser Versions: Microsoft Edge Version 93.0.961.38 (Official build) (64-bit)
Windows Version: Windows10

**Example Page URL**: https://microsoft.github.io/debug-adapter-protocol/

**Repro Steps:**

1. Open the URL: https://microsoft.github.io/debug-adapter-protocol/
2. Debug Adapter Protocol home screen will get open.
3. Start Narrator and press "D" key for checking main landmark and verify the issue.

**Actual:**

Main landmark is missing.

**Same issue is also observed with throughout the application.**

**Expected:**

Main landmark should be set to all the scenarios.

**User Impact:**

Screenreader users will not get main landmark information.

**Attachment:**

“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTag:**#A11yTCS;#DesktopWeb;#Win10;#WCAG1.3.1;#FTP;#STP;#A11yMAS;#ChromiumEdge;#A11ySev2;#P2_Visual Studio Code Clien_Sep2021;#Chrome;#Narrator;#NVDA;

**Comment by NiharikaTipirishetty** [comment]
#P2_debugadapterprotocol_Web_Sep2021;

**Comment by prabhudevu** [comment]
I have verified this issue in the Env:https://microsoft.github.io/debug-adapter-protocol/
we are not able to find the same screen that is mentioned in the bug, and we verified the issue for other links in the web paget, it is passing the color contrast ratio. added attachment for reference

![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/98821792/178416312-a19545e0-4f8c-40f0-b844-1f426ec8e1d6.png)

**Comment by prabhudevu** [comment]
#closed;

**Comment by prabhudevu** [comment]
I have verified issue in the environment: https://microsoft.github.io/debug-adapter-protocol/, Issue is fixed
added attachment for reference

![Uploading LM.png…]()
